cryptoki/tests/common.rs

```diff
@@ -11,12 +11,17 @@ pub static USER_PIN: &str = "fedcba";
 // The default SO pin
 pub static SO_PIN: &str = "abcdef";
 
+fn get_pkcs11_path() -> String {
+    env::var("PKCS11_SOFTHSM2_MODULE")
+        .unwrap_or_else(|_| "/usr/local/lib/softhsm/libsofthsm2.so".to_string())
+}
+
+pub fn is_softhsm() -> bool {
```

Comment:
Referring to my other comment, this could check which one of the env variable is set, `PKCS11_SOFTHSM2_MODULE` or `PKCS11_KRYOPTIC_MODULE`

Reply:
As mentioned in the other comment, would go ahead with something more generic rather than tying the testsuite to specific tokens. There will always have to be some skipping of tests on broken tokens, but I would like to be able to run against different (soft)tokens + modules, rather than a fixed list. Yes, we could have "default"/"other" token too, but not sure how well it would play here.